ansible/ansible-content-parser
# ValueError occurs with ansible-risk-insights >= 0.2.2

Issue: [AAP-18283](https://issues.redhat.com/browse/AAP-18283)

## Description


ansible-content-parser 0.0.7 using ansible-lint:6.22.0 ansible-core:2.15.6

ValueError: <class 'ansible_risk_insight.models.File'> is not a supported type for Sage objects

### Steps to Reproduce

ansible-content-parser --skip-ansible-lint https://github.com/IBM/Ansible-OpenShift-Provisioning.git /tmp/y

### Analysis

The class  'ansible_risk_insight.models.File' was introduced in ansible-risk-insight 0.2.2 and sage library (sage-scan) does not support the class yet.

This PR is for forcing ansible-content-parser to use ansible-risk-insight 0.2.1. Once sage-scan is updated to support the latest version of ansible-risk-insight, ansible-content-parser will be updated accordingly.


## Testing

### Steps to test
1. Pull down the PR
2. Run tox command to build a wheel (.whl) file
3. Prepare a venv and install the wheel file built on the step 2
4. Execute `ansible-content-parser --skip-ansible-lint https://github.com/IBM/Ansible-OpenShift-Provisioning.git /tmp/y` to make sure no error is displayed.

### Scenarios tested

I have run the steps described in the Steps to test section
